client/my-sites/domains/domain-search/index.jsx

```diff
@@ -236,24 +254,48 @@ class DomainSearch extends Component {
 			content = (
 				<span>
 					<div className="domain-search__content">
-						<BackButton
-							className="domain-search__go-back"
-							href={ domainManagementList( selectedSiteSlug ) }
-						>
-							<Gridicon icon="arrow-left" size={ 18 } />
-							{ translate( 'Back' ) }
-						</BackButton>
+						{ false === domainSidebarExperimentUser && (
```

Comment:
`domainSidebarExperimentUser` should be a boolean already since the selector returns a boolean, so we could simplify it.

```suggestion
						{ ! domainSidebarExperimentUser && (
```

Reply:
Good, [fixed here](https://github.com/Automattic/wp-calypso/pull/73017/commits/db39021a84115d37704b723a4f414228a102c42e#diff-e9d5bbfeb0cc388a1ec2501f08f49524a97724605eca06bc1f3f626bf12cfab4L257).